app/views/plugins/_show.haml

```diff
@@ -0,0 +1,19 @@
+- runtimeDiv = "output-#{plugin.id}"
+= content_for :external_scripts do
+  %script{src:plugin.url}
+
+.output{id:runtimeDiv}
+
+:javascript
+  // Begin script for #{plugin.name}
+  $(document).ready( function() {
+    env = {
+      name: '#{plugin.name}',
+      scriptLabel: '#{plugin.label}',
+      scriptUrl: '#{plugin.url}',
+      pluginId: '#{plugin.id}',
+      config: '#{ j plugin.author_data }',
```

Comment:
`j plugin.author_data` is it a typo or some magic Rails helper (JSON parsing)?
If so, should we agree on one common format of Plugin state (both authored and user)? I'd assume it's always `string` or `JSON`. A string can be saved in JSON and JSON can be stringified, so it doesn't matter to me. Although I'd make sure we use one format so we don't have to make checks everywhere. So far I've added user state as `string` to LARA API docs.

Reply:
Yes, this `j` is rails magic.  

It escapes text for including in javascript.  It has a better, longer, and more descriptive function name `escape_javascript` , which  I will switch to here to better document what is happening.

I will also  update the `_show.html.haml` for the ExternalScripts to use this `escape_javascript` .  I had a half-baked regex I hand-rolled previously.